scripts/templates/adapter.def.in.mako

```diff
@@ -0,0 +1,15 @@
+<%!
+import re
+from templates import helper as th
+%><%
+    n=namespace
+    N=n.upper()
+
+    x=tags['$x']
+    X=x.upper()
+%>
```

Comment:
Add a `\` at the EOL here if empty lines at the beginnings of both scripts is not intended.

Reply:
done